src/components/database/create-database/CreateDatabase.jsx

```diff
@@ -9,17 +9,21 @@ import sqlserverIcon from '../../../assets/sqlserverIcon.svg'
 import { dbEnable } from '../../../pages/database/dbActions'
 import './create-db.css'
 import { useSelector } from 'react-redux';
-import { Link } from 'react-router-dom';
+import { Link, useParams, useHistory } from 'react-router-dom';
 import RadioCard from '../../radio-card/RadioCard';
+import { getProjectConfig, setProjectConfig } from "../../../utils"
+import { validate } from 'graphql';
 
 const CreateDatabase = (props) => {
+  const history = useHistory()
   const [selectedDB, setSelectedDB] = useState(dbTypes.MONGO);
   const [alias, setAlias] = useState("mongo");
   const projects = useSelector(state => state.projects)
+  const dbconfig = getProjectConfig(projects, props.projectId, `modules.crud`)
+  const allAlias = dbconfig ? Object.entries(dbconfig).map(([alias]) => Object.assign({}, { alias: alias })) : {};
```

Comment:
Name this as dbAliasNames. Also, use Object.keys here as you just care about the key and not the value. There is also need of allAlias to be an array of objects, it can be array of string